ios/RCTCameraManager.m

```diff
@@ -123,7 +128,7 @@ - (id)init {
   if ((self = [super init])) {
 
     self.session = [AVCaptureSession new];
-    self.session.sessionPreset = AVCaptureSessionPresetHigh;
+    self.session.sessionPreset = AVCaptureSessionPresetHigh; 
```

Comment:
This is what we want to get rid of. Why provide a default value in defaultProps on the JS side if you're leaving it hard coded here?